meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/linux-renesas_%.bbappend

```diff
@@ -2,5 +2,13 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
 SRC_URI:append = " \
     file://defconfig \
+    file://0001-OF-DT-Overlay-configfs-interface-v7.patch \
```

Comment:
Is there a reason why this can't be merged into our Linux kernel git repo?

Reply:
"+"